Makefile.dep

```diff
@@ -862,6 +862,9 @@ FEATURES_USED := $(sort $(FEATURES_REQUIRED) $(filter $(FEATURES_OPTIONAL),$(FEA
 # all periph features correspond to a periph submodule
 USEMODULE += $(filter periph_%,$(FEATURES_USED))
 
+# add cpu_check address corresponding submodule
```

Comment:
please re-word: "select cpu_check_address pseudomodule if the corresponding feature is used"